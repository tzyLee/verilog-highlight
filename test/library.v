library rtlLib *.v; // matches all files in the current directory with a .v suffix
library gateLib ./*.vg; // matches all files in the current directory with a .vg suffix

include ./.../?.v;