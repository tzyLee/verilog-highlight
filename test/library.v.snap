>library rtlLib *.v; // matches all files in the current directory with a .v suffix
#^^^^^^^ source.verilog keyword.other.verilog
#       ^ source.verilog
#        ^^^^^^ source.verilog variable.other.readwrite.verilog
#              ^ source.verilog
#               ^^^ source.verilog string.other.path.verilog
#                  ^ source.verilog punctuation.terminator.statement.verilog
#                   ^ source.verilog
#                    ^^ source.verilog comment.line.double-slash.verilog punctuation.definition.comment.verilog
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.verilog comment.line.double-slash.verilog
>library gateLib ./*.vg; // matches all files in the current directory with a .vg suffix
#^^^^^^^ source.verilog keyword.other.verilog
#       ^ source.verilog
#        ^^^^^^^ source.verilog variable.other.readwrite.verilog
#               ^ source.verilog
#                ^^^^^^ source.verilog string.other.path.verilog
#                      ^ source.verilog punctuation.terminator.statement.verilog
#                       ^ source.verilog
#                        ^^ source.verilog comment.line.double-slash.verilog punctuation.definition.comment.verilog
#                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.verilog comment.line.double-slash.verilog
>
>include ./.../?.v;
#^^^^^^^ source.verilog keyword.other.verilog
#       ^ source.verilog
#        ^^^^^^^^^ source.verilog string.other.path.verilog
#                 ^ source.verilog punctuation.terminator.statement.verilog